docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md

```diff
@@ -192,7 +212,44 @@ req_body = {
 }
 ```
 
-### 2.5 Attestation
+### 2.5 Fetch and Verify Event Log
+
+CCNP provides an easy way to fetch and verify event log in TEE. After fetching event log, we can compare IMR (Integrated Measurement Register) value from CCNP fetching and replayed via event log to verify that event log has not been tampered with.
```

Comment:
we can compare the measurement replayed from event logs with the IMR (Integrated Measurement Register) values fetched using CCNP to verify that event logs have not been tampered with.